.azure-pipelines-templates/matrix.yml

```diff
@@ -1,188 +1,83 @@
 parameters:
-  consensus: ['CFT', 'BFT']
   target: ['NoSGX', 'SGX']
 
   env:
-    Checks:
+    Hosted:
       container: nosgx
-      pool: Ubuntu-1804-D16s_v3
+      pool:
+        vmImage: ubuntu-18.04
     NoSGX:
       container: nosgx
       pool: Ubuntu-1804-D16s_v3
     SGX:
       container: sgx
       pool: Ubuntu-1804-DC8_v2
-    SAN:
-      container: nosgx
-      pool: Ubuntu-1804-D16s_v3
-    Release:
-      container: sgx
-      pool: Ubuntu-1804-DC8_v2
-    Metrics:
-      container: nosgx
-      pool: Ubuntu-1804-D16s_v3
-      dependsOn: ['Test_SGX_CFT_Perf', 'Test_SGX_BFT_Perf']
-    Publish:
-      container: nosgx
-      pool: Ubuntu-1804-D16s_v3
-      dependsOn: ['Test_SGX_CFT_Release', 'Test_SGX_BFT_Release']
     Perf:
       pool: CCF-Perf
 
   build:
     common:
       cmake_args: '-DCMAKE_C_COMPILER_LAUNCHER="ccache" -DCMAKE_CXX_COMPILER_LAUNCHER="ccache"'
     NoSGX:
-      cmake_args: '-DCOMPILE_TARGETS=virtual -DCOVERAGE=ON'
+      cmake_args: '-DTARGET=virtual'
```

Comment:
Looks like a bad merge - this should still be `COMPILE_TARGETS`

Reply:
Yep, it was. Also the SGX build was really both (oops!).